dist/PublicLab.Editor.js

```diff
@@ -22758,7 +22758,12 @@ module.exports = function CustomInsert(_module, wysiwyg) {
     }
   });
   const builder = require("./PublicLab.CustomInsert.Template.js");
-  $('.wk-commands').append('<a class="woofmark-command-insert btn btn-outline-secondary" data-toggle="Insert" title="Custom Insert"><i class="fa fa-tags"></i></a>');
+  $('.wk-commands').append('<button class="woofmark-command-insert btn btn-outline-secondary" data-toggle="insert" title="Custom Insert"><i class="fa fa-tags"></i></button>');
+
+  $(document).ready(function() {
+    $('[data-toggle="insert"]').tooltip();
+  });
+
```

Comment:
Hi @RaviAnand111, this change isn't a part of this PR.

Reply:
@RaviAnand111 Only remove this change from #800 but leave it here. The reason why this change keeps showing up is because `grunt build` was not run before opening this PR #782

Reply:
I did run `grunt build` before making the PR actually, 
I don't understand whats going wrong here ?

Reply:
@RaviAnand111 what @TildaDares is trying to say is that the PR that added those lines of code:
```
  $(document).ready(function() {
    $('[data-toggle="insert"]').tooltip();
  });
```
didn't run `grunt build` before creating the PR and the PR is #782
So please follow @TildaDares guide and try again. 
You got this!

Reply:
ohh, now I understand , ok

Reply:
I just have to remove these lines from my PR #800  right @NARUDESIGNS @TildaDares and not from this commit right ?

Reply:
Exactly @RaviAnand111

Reply:
ok, Thanks @TildaDares you are awesome

Reply:
I have done as you asked, can you please check it, 
ThankYou